dcs/action.py

```diff
@@ -1787,6 +1787,25 @@ def dict(self):
         return d
 
 
+class PictureToAll(Action):
+    predicate = "a_out_picture"
+
+    def __init__(self, file_res_key: Optional[ResourceKey] = None):
+        super(PictureToAll, self).__init__(PictureToAll.predicate)
+        if file_res_key:
+            self.file_res_key = file_res_key
+            self.params.append(self.file_res_key)
+
+    @classmethod
+    def create_from_dict(cls, d, mission):
+        return cls(ResourceKey(d["file"]))
```

Comment:
Based on how you've written the rest of the class, this looks wrong. Is the `ResourceKey` optional or not?